platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/InputValidationTest.java

```diff
@@ -44,14 +46,21 @@
 
     private static final String BAD_MIME = "text/html";
 
+    private static final String IGNORE_FILE = ".DS_Store";
+
+    private static final String IGNORE_FILE_CASE_INSENSITIVE = ".dS_sTorE";
+
+    private static final String IGNORE_FILE_KNOWN_GOOD = "valid_ddms_record.xml";
+
     @Before
     public void setup() {
         System.setProperty("bad.files",
```

Comment:
I recommend using this library to manage system properties for testing. 
http://stefanbirkner.github.io/system-rules/

Using it ensures that the system properties are restored to their original values.